backend/app/api/v1/commons/utils.py

```diff
@@ -103,7 +105,11 @@ def buildReleaseStreamFilter(input_array):
     for item in input_array:
         # Find the first matching key in the map
         match = next(
-            (value for key, value in RELEASE_STREAM_DICT.items() if key in item),
+            (
+                value
+                for key, value in constants.RELEASE_STREAM_DICT.items()
+                if key in item
+            ),
```

Comment:
As in the other PR, changing value and key to just v and k might make this easier to read.

Reply:
Sadly it didn't change the formatting? That's too bad, but I'll resolve the conversation anyway.